### PR TITLE
Gameplay startup

### PR DESCRIPTION
**Pull request by mdesson**
Gameplay startup. Let me know if there's any issues.

### CODE REVIEW

File: src/GameStartup/GameStartupDriver.cpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+void gameStartupExample()
+{
+  GameState state();
+  std::cout << "When the game state is created, the deck is shuffled.\n";
+  std::cout << "At the same time 6 cards are drawn.\n";
+  std::cout << "In the two game states below, the cards are shuffled different.\n";
+  std::cout << "They are drawn from the top of shuffled deck and are therefore different.\n";
+  GameState state1 = GameState();
+  GameState state2 = GameState();
```
Comment (seanheinrichs) [inline src/GameStartup/GameStartupDriver.cpp:15]: What's the second state for?

File: src/Map/Map.h [hunk 2]
```diff
@@ -43,6 +44,9 @@ class Map
     // print details of each node
     void printNodes();
 
+    // get name of starting city
```
Comment (seanheinrichs) [inline src/Map/Map.h]: starting region, not city right? I guess its technically a city though xD

**Review by seanheinrichs — COMMENTED** [review]
whelp, didnt run map 8. ignore this!

**Review by seanheinrichs — APPROVED** [review]

File: src/GameStartup/GameStartupDriver.cpp (hunk 1 above)
Comment (mdesson) [reply src/GameStartup/GameStartupDriver.cpp:15]: Testing purproses! To show that they have different draw piles.

File: src/Map/Map.h (hunk 2 above)
Comment (mdesson) [reply src/Map/Map.h]: Good catch, fixed.

**Pull request merged**